warp-contracts-plugin-signature/src/web/evm/evmSignature.ts

```diff
@@ -1,60 +1,42 @@
+import MetaMaskOnboarding from '@metamask/onboarding';
 import { MetaMaskInpageProvider } from '@metamask/providers';
 import Transaction from 'arweave/web/lib/transaction';
-import { encodeTxId } from '../utils';
-import MetaMaskOnboarding from '@metamask/onboarding';
-import { stringify } from 'safe-stable-stringify';
 import { utils } from 'ethers';
-import { SmartWeaveTags, TagsParser } from 'warp-contracts/web';
-import { Interaction } from './evmSignatureVerification';
+import { attachSignature, prepareEvmTransaction } from './common';
+
 declare global {
   interface Window {
     ethereum?: MetaMaskInpageProvider;
   }
 }
 
-export const evmSignature = async (tx: Transaction): Promise<void> => {
-  const tagsParser = new TagsParser();
+export const evmSignature = async (tx: Transaction) => {
```

Comment:
I think that maybe name like `metaMaskSignature` could be more precise, however I didn't want to break contract.